modules/gapi/include/opencv2/gapi/s11n.hpp

```diff
@@ -225,6 +292,23 @@ cv::GCompileArgs getCompileArgs(const std::vector<char> &p) {
 
     return args;
 }
+
+template<typename RMatAdapterType>
+cv::GRunArgs getRunArgsWithRMats(const std::vector<char> &p) {
+    std::unique_ptr<cv::gapi::s11n::IIStream> pIs = cv::gapi::s11n::detail::getInStream(p);
+    cv::gapi::s11n::IIStream& is = *pIs;
+    cv::GRunArgs args;
+
+    uint32_t sz = 0;
+    is >> sz;
+    for (uint32_t i = 0; i < sz; ++i) {
+        uint32_t idx = 0;
+        is >> idx;
```

Comment:
This is wrong, since eventually I will try to do the same in variant serialization